tests/org.eclipse.passage.lbc.base.tests/src/org/eclipse/passage/lbc/base/tests/EncryptTestKeys.java

```diff
@@ -0,0 +1,67 @@
+package org.eclipse.passage.lbc.base.tests;
```

Comment:
Can you be so kind to explain the purpose of this?
This is neither _final_ nor _abstract_ class, full of static methods and, surprisingly, posessing _main_. For what I see, it's unused. If you meant some testing necessities to be satisfied here, that would be great to convey them into accordance with coding principles of the project.

Reply:
I used this code to create encrypt the modified `.lic` to get the new `.licen` file. That is the only purpose and the reason it has a main. It is not needed to execute the tests.
I thought you might be interested, in case some license files used for testing have to be changed and encrypted again.
But I'm also fine with removing it from this PR.